### PR TITLE
2798: Adding Total XP Earnings Tracking

### LINKED ISSUE #2798: overall XP tracked for personnel.
title says it all.. would be nice to have the xp accumulated instead of disappearing after being the xp is used

KG

### PR DESCRIPTION
**Pull request by Windchild292**
This fixes #2798, pretty much copied the logic for Track Total Earnings when doing this.

### CODE REVIEW

File: MekHQ/src/mekhq/campaign/finances/Finances.java [hunk 1]
```diff
@@ -331,7 +331,7 @@ public void newDay(Campaign campaign) {
                             String.format(resourceMap.getString("Salaries.text"),
                                     payRollCost.toAmountAndSymbolString()));
 
-                    if (campaign.getCampaignOptions().trackTotalEarnings()) {
+                    if (campaign.getCampaignOptions().isTrackTotalEarnings()) {
```
Comment (NickAragua) [inline MekHQ/src/mekhq/campaign/finances/Finances.java:334]: In terms of english language syntax, it's pretty awkward.

**Review by NickAragua — APPROVED** [review]
Other than awkward english syntax in some variable names, looks fine.

File: MekHQ/src/mekhq/campaign/finances/Finances.java (hunk 1 above)
Comment (Windchild292) [reply MekHQ/src/mekhq/campaign/finances/Finances.java:334]: It's a style thing... is means it's a boolean value, otherwise get is used.

Comment (NickAragua) [reply MekHQ/src/mekhq/campaign/finances/Finances.java:334]: Sure, but when you're reading it in English, the sentence structure is pretty awkward.

**Pull request merged**